manager/src/test/java/com/redhat/service/bridge/manager/workers/resources/BridgeWorkerTest.java

```diff
@@ -98,7 +103,7 @@ void handleWorkProvisioningWithKnownResourceWithConnector(ManagedResourceStatus
         when(bridgeDAO.findById(RESOURCE_ID)).thenReturn(bridge);
         when(bridgeDAO.getEntityManager()).thenReturn(entityManager);
         when(entityManager.merge(bridge)).thenReturn(bridge);
-        when(internalKafkaConfigurationProvider.getBridgeTopicName(bridge)).thenReturn(TOPIC_NAME);
+        when(resourceNamesProvider.getBridgeTopicName(bridge.getId())).thenReturn(TOPIC_NAME);
```

Comment:
Can we avoid the mock in here?

Reply:
Given the way the test is run the short answer is no. I think we already discussed to limit the use of `MockitoExtension` based tests, so we'll refactor this later on if needed.